### PR TITLE
Add new contributor profile

### PR DESCRIPTION
**Pull request by howl-anderson**
ref: wechaty/summer#87

### CODE REVIEW

File: jekyll/_contributors/howl-anderson.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Xiaoquan Kong
+site: https://github.com/howl-anderson
+avatar: /assets/contributors/howl-anderson/avatar.jpeg
```
Comment (lijiarui) [inline jekyll/_contributors/howl-anderson.md]: Please change the image format as GitHub Action required.

File: jekyll/_contributors/howl-anderson.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+bio: Google Developer Expert (GDE) for Machine Learning. TensorFlow Addons codeowner. SuperHero contributor for Rasa. Author and translator of books.
+---
+
```
Comment (lijiarui) [inline jekyll/_contributors/howl-anderson.md:7]: Please add your PR link as well, thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_contributors/howl-anderson.md (hunk 1 above)
Comment (howl-anderson) [reply jekyll/_contributors/howl-anderson.md]: Fixed

File: jekyll/_contributors/howl-anderson.md (hunk 2 above)
Comment (howl-anderson) [reply jekyll/_contributors/howl-anderson.md:7]: I will add a new section for PRs in the profile in the next commit

**Review by lijiarui — APPROVED** [review]
Cheers！

Welcome to join Wechaty Commiunity!

**Pull request merged**